### PR TITLE
Cherry-pick #2402

### PR DESCRIPTION
**Pull request by ashfurrow**
This PR cherry-picks commits #2402 (excluding a merge from master) onto the `3.2.3-release` branch. See [Slack](https://artsy.slack.com/archives/C02BAQ5K7/p1504720100000277) for more details.

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">You need to set the App’s plist version to 3.3.0</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_travis">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by ashfurrow** [comment]
Failing Danger check is okay – this isn't a normal PR. I'll deal with that afterward.

File: CHANGELOG.yml [hunk 1]
```diff
@@ -1,5 +1,26 @@
 upcoming:
     version: 3.2.3
+    version: 3.3.0
```
Comment (orta) [inline CHANGELOG.yml]: this is a bad merge

File: Podfile.lock [hunk 2]
```diff
@@ -431,9 +431,9 @@ SPEC CHECKSUMS:
   AFOAuth1Client: 07ccc935ba06ac8d023c16d39a5bdf04bc6f92e1
   ALPValidator: c74ea0d49bbbff0f8a4228f1eb6589b992255cfe
   Analytics: b342fb4f43fa4f97ca6f4358b44d3295217ba294
-  AppHub: '03788112dd48c42b60d51321c0ffff4ef15a4432'
-  ARAnalytics: 96e23ba7f54298f4914de9e1ac999db7a15f3fdd
-  ARCollectionViewMasonryLayout: 776730bdedd0c7570a5e904c370e4e120f98ea62
+  AppHub: 03788112dd48c42b60d51321c0ffff4ef15a4432
```
Comment (orta) [inline Podfile.lock]: lolz quotes

**Comment by orta** [comment]
OK, cool 👍  (I'm assuming the duped YAML overwrites itself)

File: CHANGELOG.yml (hunk 1 above)
Comment (ashfurrow) [reply CHANGELOG.yml]: Just caught your comment – aye, I'll fix for posterity.

**Pull request merged**